packages/amplify-category-notifications/package.json

```diff
@@ -22,7 +22,7 @@
     "watch": "tsc --watch"
   },
   "dependencies": {
-    "amplify-cli-core": "3.2.0",
+    "amplify-cli-core": "3.2.1",
```

Comment:
is this the only place that needs updated?

Reply:
yeah from this got skipped when resolving merge conflicts to dev.
In rest places we are using amplify-cli-core: 3.2.1